types/okta__okta-vue/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for okta-vue 1.1
+// Type definitions for okta-vue 1.2
 // Project: https://github.com/okta/okta-oidc-js/tree/master/packages/okta-vue, https://github.com/okta/okta-oidc-js
 // Definitions by: Mike Dodge <https://github.com/innovation-team>
```

Comment:
@mgdodge is this you? You should update/add your GitHub account so future PRs get fast-tracked.

Reply:
Yes, this is @mgdodge. Not sure what you mean by updating/adding my github account. The changes/updates to this library go through a company account since all my usage is on company projects, that's why everything comes from innovation-team.

Reply:
You made this change through @mgdodge, but this header lists @innovation-team as the owner. You should add @mgdodge to the list so that when you make PRs from that account in the future, @typescript-bot will fast-track the review and you won’t have to wait as long. Also if someone else makes a PR to these definitions, it will ping you (as in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40944#issuecomment-564140263).

```suggestion
// Definitions by: Mike Dodge <https://github.com/mgdodge>
```